server/src/main/java/org/bonitasoft/console/common/server/servlet/FileUploadServlet.java

```diff
@@ -171,6 +174,15 @@ public void doPost(final HttpServletRequest request, final HttpServletResponse r
                 responsePW.print(responseString);
                 responsePW.flush();
             }
+        } catch (final FileTooBigException e) {
+            if (LOGGER.isLoggable(Level.SEVERE)) {
+                LOGGER.log(Level.SEVERE, "File is Too Big", e);
+            }
+            response.setStatus(HttpURLConnection.HTTP_ENTITY_TOO_LARGE);
+            if (JSON_CONTENT_TYPE.equals(responseContentType)) {
+                responsePW.print("{\"type\":\"error\", \"message\": \"" + e.getFileName() + " is too big, limit is set to " + e.getMaxSize() + "Mb\"}");
```

Comment:
Hu... Don't we have jackson capabilities here ?
IMHO as error type, we should put something like `FileTooBig` so user can display specific messages for that